vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -52,15 +52,20 @@ def get_min_capability(cls) -> int:
     def get_name(self) -> str:
         return "compressed_tensors"
 
-    # TODO (@robertgshaw2-neuralmagic): do layer skipping though here
-    # rather than though create_weights to match other methods
     def get_quant_method(
         self,
         layer: torch.nn.Module,
         prefix: str,
-    ) -> Optional["QuantizeMethodBase"]:
+    ) -> Optional[Union["QuantizeMethodBase"]]:
```

Comment:
Why is Union needed here for a single type?

Reply:
Forgot to remove. Good catch